<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QDialog" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Update dB</string>
  </property>
  <widget class="QTextEdit" name="cin">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="get_cin">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>60</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>RÃ©cuperez</string>
   </property>
  </widget>
  <widget class="QLabel" name="etudiant">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Etudiant</string>
   </property>
  </widget>
  <widget class="QLabel" name="student">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>110</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>421</width>
     <height>201</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Initiation</string>
    </attribute>
    <widget class="QLineEdit" name="note_initiation">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>20</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Perfectionnement</string>
    </attribute>
    <widget class="QLineEdit" name="note_perfectionnement">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>351</width>
       <height>121</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Attestation</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="presente">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>140</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Presente</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="manquante">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>140</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Manquante</string>
   </property>
  </widget>
  <widget class="QPushButton" name="edit_db">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>60</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Modifiez</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
